ccd-definition/CaseEventToFields/CareSupervision/submitApplication.json

```diff
@@ -21,7 +21,8 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/case-submission/about-to-start"
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/case-submission/about-to-start",
```

Comment:
Any reason why we have an about to start event defined here? I don't think it will even be pulled into the spreadsheet

Reply:
Would also double check if all the fields in this event are required on the CYA page

Reply:
have removed label but left declaration. not sure on about-to-start call back as it was defined before - will remove and check if working though as yeah don't think it would either